en/docs/install-and-setup/setup/security/enabling-hostname-verification.md

```diff
@@ -16,6 +16,21 @@ The hostname verification is **disabled** by default. This is done using the `o
     -Dhttpclient.hostnameVerifier=AllowAll \
 ```
 
+When applying the strict hostname verification, you need to change the configuration in the `site/public/conf/settings.js` file of each webapp - Publisher, Developer Portal, and Admin.
```

Comment:
Let's give the path clearly here as well.

Reply:
Fixed.